app/code/Magento/Ui/view/base/web/js/form/form.js

```diff
@@ -275,14 +276,29 @@ define([
          */
         focusInvalid: function () {
             var invalidField = _.find(this.delegate('checkInvalid'));
-
             if (!_.isUndefined(invalidField) && _.isFunction(invalidField.focused)) {
+                this.focusInvalidOnCollapsibleOpen(invalidField);
                 invalidField.focused(true);
             }
-
             return this;
         },
 
+        /**
+         * Add an event listener to the collapsible content block which triggers
+         * invalid input focusing when collapsible is open.
+         *
+         * @param {Object} invalidField
+         */
+        focusInvalidOnCollapsibleOpen: function (invalidField) {
+            var collapsibleContent = $('#' + invalidField.uid).closest(this.collapsibleSelector);
+            if (collapsibleContent.length > 0) {
+                collapsibleContent.on('transitionend', () => {
```

Comment:
Looks like a new listener will be set each time when `focusInvalidOnCollapsibleOpen` will be called. But if I right understand we need to do it just once